lib/cb/version.rb

```diff
@@ -1,3 +1,3 @@
 module Cb
-  VERSION = '16.2.2'
+  VERSION = '17.0.0'
```

Comment:
It is still returning a string. Just one that is missing the array markup. 
before 
"[\"File upload data is larger than the maximum of 1024KB.\"]"

now
"File upload data is larger than the maximum of 1024KB.\"